### PR TITLE
Doc: Reading GTFS specs (WIP)

### PR DESCRIPTION
**Pull request by prhod**

### CODE REVIEW

File: src/documentation/gtfs2ntfs.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+## How to use
+### Basic Use
+A basic usage of gtfs2ntfs is the following : 
+`./gtfs2ntfs -i path/to/gtfs.zip -o path/to/dest/ntfs.zip`
+
+This will convert the GTFS into a valid NTFS, using the conversion method described in [gtfs_read.md](./gtfs_read.md) specifications. See next chapter to details about this use.
+
+### Use with a config file
+A mode complexe read method is to provide a config file that will contain details about :
```
Comment (datanel) [inline src/documentation/gtfs2ntfs.md]: A **more complex**?

Comment (datanel) [reply src/documentation/gtfs2ntfs.md]: in english there is no space before **:**

File: src/documentation/gtfs_read.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+### Reading trips.txt 
+
+| NTFS file | NTFS field | Constraint | GTFS file | GTFS field | Note |
+| --- | --- | --- | --- | --- | --- |
+| trips.txt | route_id | Required | trips.txt | route_id | cf. NTFS `route_id` definition above to specify the propert reference. |
+| trips.txt | service_id | Required | trips.txt | service_id |  |
+| trips.txt | trip_id | Required | trips.txt | trip_id |  |
+| trips.txt | trip_headsign | Optionnal | trips.txt |  | `trip_short_name`, of if empty `trip_headsign` |
+| trips.txt | block_id | Optionnal | trips.txt | block_id |  |
+| trips.txt | company_id | Required | trips.txt | agency_id |  |
```
Comment (datanel) [inline src/documentation/gtfs_read.md]: agency_id from route, not trip

**Comment by prhod** [comment]
GTFS reading specs are written until stop_times. It's ok for review/merge, even though it's partial.
Calendars (and the rest) will be done later.

**Pull request merged**